android/tangram/jni/platform_android.cpp

```diff
@@ -243,5 +246,27 @@ void setCurrentThreadPriority(int priority) {
     setpriority(PRIO_PROCESS, tid, priority);
 }
 
+void featureSelectionCallback(const std::vector<Tangram::TouchItem>& items) {
+
+    JNIEnv *jniEnv;
```

Comment:
Right, this can now be changed to use the jnienv passed into Java_com_mapzen_tangram_MapController_pickFeature (the only caller of featureSelectionCallback).  As far as I know java threads are native threads as well on Android. 


As a side-note: I noticed that there there are two calls to init / setupJniEnv: once on the main-thread and once on androids render-thread.